### PR TITLE
Revert profile base URL behavior

### LINKED ISSUE #8172: Pinned Post feature broken
### Expected behavior
Pinned posts should be pinned. The option in the action menu should toggle the pin/unpin status.

### Actual behavior
My pinned posts are not pinned anymore. It is not possible to pin them again or to pin other posts. This is probably caused by PR #8156 because it worked before I pulled it.

### Steps to reproduce the problem
Check your pinned posts on your wall/status page. Try to toggle the pinned status.

### Friendica version you encountered the problem
2020.03-dev

### Friendica source (git, zip)
git

### PHP version
7.2

### SQL version
10.1 MariaDB

### PR DESCRIPTION
**Pull request by MrPetovan**
Addresses https://github.com/friendica/friendica/pull/8156#issuecomment-578467641
Fixes #8172

### CODE REVIEW

File: mod/cal.php [hunk 1]
```diff
@@ -121,7 +121,7 @@ function cal_content(App $a)
 	$sql_extra = " AND `event`.`cid` = 0 " . $sql_perms;
 
 	// get the tab navigation bar
-	$tabs = Profile::getTabs($a, 'cal', false, $a->data['user']['nickname']);
+	$tabs = Friendica\Module\BaseProfile::getTabsHTML($a, 'cal', false, $a->data['user']['nickname']);
```
Comment (nupplaphil) [inline mod/cal.php]: Plz Import the namespace for better readability.

File: mod/events.php [hunk 2]
```diff
@@ -251,7 +251,7 @@ function events_content(App $a)
 	$tabs = '';
 	// tabs
 	if ($a->theme_events_in_profile) {
-		$tabs = Profile::getTabs($a, 'events', true);
+		$tabs = Friendica\Module\BaseProfile::getTabsHTML($a, 'events', true);
```
Comment (nupplaphil) [inline mod/events.php]: Plz Import the namespace for better readability.

File: mod/notes.php [hunk 3]
```diff
@@ -28,7 +28,7 @@ function notes_content(App $a, $update = false)
 		return;
 	}
 
-	$o = Profile::getTabs($a, 'notes', true);
+	$o = Friendica\Module\BaseProfile::getTabsHTML($a, 'notes', true);
```
Comment (nupplaphil) [inline mod/notes.php]: Plz Import the namespace for better readability.

File: mod/photos.php [hunk 4]
```diff
@@ -914,7 +914,7 @@ function photos_content(App $a)
 
 	// tabs
 	$is_owner = (local_user() && (local_user() == $owner_uid));
-	$o .= Profile::getTabs($a, 'photos', $is_owner, $a->data['user']['nickname']);
+	$o .= Friendica\Module\BaseProfile::getTabsHTML($a, 'photos', $is_owner, $a->data['user']['nickname']);
```
Comment (nupplaphil) [inline mod/photos.php]: Plz Import the namespace for better readability.

File: src/Module/BaseProfile.php [hunk 5]
```diff
@@ -0,0 +1,127 @@
...
+
+use Friendica\App;
+use Friendica\BaseModule;
+use Friendica\Core\Hook;
+use Friendica\Core\Renderer;
+use Friendica\DI;
+
+class BaseProfile extends BaseModule
+{
+	/**
```
Comment (nupplaphil) [inline src/Module/BaseProfile.php]: Maybe you can add a short summary about the method :) a sentence would be enough I guess ^^

File: src/Module/BaseProfile.php [hunk 6]
```diff
@@ -0,0 +1,127 @@
...
+				'label' => DI::l10n()->t('Contacts'),
+				'url'   => $baseProfileUrl . '/contacts',
+				'sel'   => $current == 'contacts' ? 'active' : '',
+				'title' => DI::l10n()->t('Contacts'),
+				'id'    => 'viewcontacts-tab',
+				'accesskey' => 'k',
+			];
+		}
+
+		if (!empty($_SESSION['new_member']) && $is_owner) {
```
Comment (nupplaphil) [inline src/Module/BaseProfile.php]: `DI::session()->get('new_member')`

File: src/Module/BaseProfile.php (hunk 5 above)
Comment (MrPetovan) [reply src/Module/BaseProfile.php]: It's going to be so redundant with the name of the method you will be twice sorry you asked. 😛

**Pull request merged**